### ISSUE TITLE
exported external package typings file is not a module

### ISSUE THREAD

**Issue by Mihai-B** [cstefanache/angular2-img-cropper#4]
I am trying to use this library with angular 2 rc1 and angular-cli beta 5.

This is what I have done right now:
in angular-cli-build.js I have added` 'ng2-img-cropper/dist/*.+(d.ts|js|js.map)'`

In system-config.ts: 
```
const map: any = {
  'ng2-img-cropper': 'vendor/ng2-img-cropper'
};
```
and:
```
const packages: any = {
  'ng2-img-cropper/dist': {
    format: 'cjs',
    defaultExtension: 'js',
    main: 'ng2-img-cropper.js'
  }

```
I import it in the file where I use it: import {ImageCropperComponent, CropperSettings} from 'ng2-img-cropper/dist/ng2-img-cropper';

but wen I want to build the file I get:

```
Typescript found the following errors:
  /home/mike/dev/myProj/tmp/broccoli_type_script_compiler-input_base_path-n1z8vOxo.tmp/0/src/app/components/project/cropp-project/cropp-project.component.ts (2, 54): File '/home/mike/devmyProj/node_modules/ng2-img-cropper/dist/ng2-img-cropper.d.ts' is not a module.
  /home/mike/dev/myProj/tmp/broccoli_type_script_compiler-input_base_path-n1z8vOxo.tmp/0/src/app/components/project/cropp-project/cropp-project.component.ts (2, 54): Exported external package typings file '/home/mike/dev/myProj/node_modules/ng2-img-cropper/dist/ng2-img-cropper.d.ts' is not a module. Please contact the package author to update the package definition.
    at BroccoliTypeScriptCompiler._doIncrementalBuild (/home/mike/devmyProj/node_modules/angular-cli/lib/broccoli/broccoli-typescript.js:115:19)
    at BroccoliTypeScriptCompiler.build (/home/mike/dev/myProj/node_modules/angular-cli/lib/broccoli/broccoli-typescript.js:43:10)
    at /home/mike/dev/myProjb/node_modules/broccoli-caching-writer/index.js:152:21
    at lib$rsvp$$internal$$tryCatch (/home/mike/dev/myProj/node_modules/rsvp/dist/rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (/home/mike/dev/myProj/node_modules/rsvp/dist/rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (/home/mike/dev/myProj/node_modules/rsvp/dist/rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (/home/mike/dev/myProj/node_modules/rsvp/dist/rsvp.js:1198:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)

```

Am I doing something wrong? Or this is a bug?

**Comment by cstefanache** [comment]
Might be a packaging issue. I will try to investigate ASAP.

**Comment by Mihai-B** [comment]
Any progress on this issue?

**Comment by cstefanache** [comment]
Started to recreate the packages yesterday. Will probably be published today -> Package will suffer changes but source code will remain the same

**Comment by cstefanache** [comment]
I just published a new package (0.3.7) on the npm with the latest changes.
I used commonjs for library packaging and created a index.d and index.tx (similar how @angular applications are packaged) so they can be imported similarly. 

Can you please make another test?

**Comment by Mihai-B** [comment]
Yes, it works, issue is now fixed
